tests/ports/unix/extra_coverage.py

```diff
@@ -33,6 +33,7 @@
 print(stream.read(1))  # read 1 byte encounters non-blocking error
 print(stream.readline())  # readline encounters non-blocking error
 print(stream.readinto(bytearray(10)))  # readinto encounters non-blocking error
+print(stream.readinto1(bytearray(10)))  # call readinto1
```

Comment:
Comment is stating the obvious and makes it less clear that it should also have the same non-blocking "error" as the other calls around it.

Reply:
You're right.

I originally added this just to call `readinto1()` regardless of the behaviour, just to get coverage of this function.  But it's obviously confusing and wrong.

I've now updated this comment, and added additional tests for readinto/readinto1 which are supposed to test the difference between the two.